multinode_integration_tests/src/masq_real_node.rs

```diff
@@ -1289,6 +1305,7 @@ mod tests {
 
     #[test]
     fn node_startup_config_builder_settings() {
+        let min_hops_count = Hops::SixHops;
```

Comment:
If you `use`d `SixHops` the way you did `TwoHops` above, you'd be able to just say `SixHops` here instead of `Hops::SixHops`.

Or you could go the other way, remove the `use` for `TwoHops`, and qualify both of them with `Hops::` if you think it's clearer.

Reply:
I preferred to remove the `use` for `TwoHops` and use `Hops::` instead.